src/shell/dotnet/Shell/Layout/LayoutManager.cs

```diff
@@ -205,15 +217,25 @@ private async Task LoadModulesAsync()
 
                     foreach (var moduleId in moduleStates.Keys)
                     {
-                        var tcs = new TaskCompletionSource<bool>();
-                        _moduleReadyTasks[moduleId] = tcs;
-
-                        await _moduleLoader.StartModule(new StartRequest(moduleStates[moduleId], new Dictionary<string, string>
-                    {
-                        { SerializationId, moduleId }
-                    }));
-
-                        moduleTasks.Add(tcs.Task);
+                        try
```

Comment:
I'm not convinced this does what you intended it to do - wouldn't this still fail if the individual tasks failed?

Reply:
I have reverted back :)